cmd/version.go

```diff
@@ -81,3 +87,38 @@ func getPluginVersions(opts Options) []string {
 
 	return versions
 }
+
+// Checks GitHub releases and prints new version, if current version is outdated.
+// requires GitHub releases to follow semver.
+func (cli *CLI) printLatestReleaseVersion() {
+	latest, err := getLatestVersion()
+	if err != nil {
+		cli.formatter.Print(tflint.Issues{}, fmt.Errorf("Failed to check updates; %w", err), map[string][]byte{})
+	}
+	latestVersion, err := version.NewSemver(*latest.Name)
+	if err != nil {
+		cli.formatter.Print(tflint.Issues{}, fmt.Errorf("Failed to parse version; %w", err), map[string][]byte{})
+	}
+	compare := tflint.Version.Compare(latestVersion)
+	if compare < 0 {
+		fmt.Fprintf(cli.outStream, "New version available: %s\n", *latest.HTMLURL)
+	}
+}
+
+func getLatestVersion() (*github.RepositoryRelease, error) {
+	ghClient := github.NewClient(nil)
+	releases, _, err := ghClient.Repositories.ListReleases(context.Background(),
```

Comment:
This should use `/releases/latest` and not list releases:

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release

That will handle excluding prereleases and drafts. Alternatively, if we had to use list, we should be fetching smaller pages than the 30 returned by default.